needy/project.py

```diff
@@ -75,6 +75,10 @@ def build_concurrency(self):
             concurrency = min(concurrency, self.configuration('max-concurrency'))
         return concurrency
 
+    def build_targets(self):
+        targets = self.configuration('build-targets')
```

Comment:
I thought about that, but I believe it might be a little ambiguous if this is also designed to allow for cross-compilation.